R/zzz.R

```diff
@@ -65,7 +65,7 @@ print_tmlenet_opts <- function() {
 #' @return Invisibly returns a list with old option settings.
 #' @seealso \code{\link{print_tmlenet_opts}}
 #' @export
-tmlenet_options <- function(useglm = FALSE,
+tmlenet_options <- function(bin_estimator = speedglmR6$new(),

```

Comment:
I'm not sure how many people are depending on / using this package already, but by doing this we do change the external interface of the package. We could always choose to go with a hybrid approach, in which we keep supporting the old API, but also extend it to support the new one (I'm not sure if we'd want to create this kind of legacy already, but then again, I don't know how many people use it). Another option is to just go for this api, and make it very clear in some release notes that we've changed this.